src/ts/InternalBitmovinYospacePlayer.ts

```diff
@@ -455,15 +455,18 @@ export class InternalBitmovinYospacePlayer implements BitmovinYospacePlayerAPI {
     if (this.adImmune) {
       const currentTime = this.player.getCurrentTime();
 
-      this.session?.getAdBreaksByType(BreakType.LINEAR).forEach((adBreak) => {
-        const breakStart = this.toMagicTime(toSeconds(adBreak.getStart()));
+      const linearAdBreaks = this.session?.getAdBreaksByType(BreakType.LINEAR);
+      if (linearAdBreaks !== null) {
```

Comment:
`linearAdBreaks` can also be `undefined` (if `this.session` is null/undefined).

This checks for a "truthy" value instead of specifically for `null`:

```suggestion
      if (linearAdBreaks) {
```

Reply:
thank you; removed the if completely and kept the ?. operator instead.